### DISCUSSION TITLE
Installing LSPosed on Pixel 6 disables Zygisk

### DISCUSSION THREAD

**Discussion by joshoram80** [LSPosed/LSPosed#1390] [Q&A] [answered]
I'm have installed the 1.6.3-6214 Zygisk build and after reboot, Zygisk is disabled in Magisk(latest canary build). If I remove the module and reboot, Zygisk is enabled again.  I have tried disabling and enabling Zygisk with the module installed but it won't turn on.

**Comment by yujincheng08** [comment, marked as answer]
Zygisk has a concession strategy. That means if a module is crashing for serval times, Zygisk will be disabled to avoid bootloop. So you have to open an issue with log.
